cf-execd/exec-config.c

```diff
@@ -51,6 +51,59 @@ static char *GetIpAddresses(const EvalContext *ctx)
     return StringWriterClose(ipbuf);
 }
 
+static void RegexFree(void *ptr)
+{
+    pcre_free(ptr);
+}
+
+static void MailFilterFill(const void *input, bool rlist,
```

Comment:
I split this into three functions, one for each of the `Seq` and `Rlist` parts, and one common one.